doc/api/http.md

```diff
@@ -2143,7 +2143,7 @@ the request body should be sent. See the [`'checkContinue'`][] event on
 added: REPLACEME
```

Comment:
I'd include `changes` tagging this PR.

Reply:
Just updated the pull request. Thank you for the review.